script/script.js

```diff
@@ -193,8 +344,36 @@ webix.ready(function () {
     webix.ui({
         rows: [
             toolbar,
-            main,
+            {
+                cols: [sideMenu, resizer, main]
+            },
             footer
         ]
     });
+    $$("list").select("dashboard");
+
+    // filtering user's data
+    const usersList = $$("usersList");
+    $$("filterUsersList").attachEvent("onTimedKeypress", function () {
+        const filterText = this.getValue().toLowerCase();
+
+        usersList.filter(function (obj) {
+            const strForSearch = (obj.name + obj.country).toLowerCase();
+            return (strForSearch.indexOf(filterText) !== -1);
+        });
+    });
+
+    // sorting users by name ascending
+    $$("sortAscBtn").attachEvent("onItemClick", function () {
+        usersList.sort("#name#", "asc", "string");
+        // usersList.sort("#age#", "asc", "int");
+    });
+
+    // sorting users by name descending
+    $$("sortDescBtn").attachEvent("onItemClick", function () {
```

Comment:
the same - move event inside of button config

Reply:
done